tests/testthat/test-as_curve_params.R

```diff
@@ -0,0 +1,25 @@
+test_that("`as_curve_params()` produces an error when non-curve data is provided", {
+  library(magrittr)
+  expect_error(
+    object = curve_data <-
+      "https://osf.io/download//n6cp3/" %>% # pop data
+      readr::read_rds() %>%
+      as_curve_params(),
+    class = "not curve_params"
+  )
+})
+
+
+test_that("`as_curve_params()` produces expected results", {
+  library(dplyr)
+  test_data <- "https://osf.io/download/rtw5k/" %>% # curve data
```

Comment:
`test_data <-` on line 15 still should be deleted; please see my previous comment about why assigning the result to a variable causes problems for this test. Another option would be to add `print(test_data)` afer line 18.